optuna/importance/_mutual_information/_parzen_estimator.py

```diff
@@ -0,0 +1,390 @@
+from typing import Callable
+from typing import Dict
+from typing import NamedTuple
+from typing import Optional
+from typing import Tuple
+
+import numpy as np
+import scipy.special
+import scipy.stats
+
+from optuna import distributions
+from optuna.distributions import BaseDistribution
```

Comment:
[question] why do you import only BaseDistribution while not the other concrete distribution classes?

Reply:
I did not do this intentionally. Probably, I imported `BaseDistribution` because it is used for type hints. Lengthy type hints are not easy to read. What do you think is the best practice in this situation? Should all concrete distribution classes be imported?